subprojects/gradle/lint-report/src/main/kotlin/com/avito/android/lint/LintSlackReporter.kt

```diff
@@ -0,0 +1,65 @@
+package com.avito.android.lint
+
+import com.avito.slack.SlackClient
+import com.avito.slack.model.SlackChannel
+import com.avito.utils.logging.CILogger
+
+interface LintSlackReporter {
+
+    fun report(
+        models: List<LintReportModel>,
+        channel: SlackChannel
+    )
+
+    class Impl(
+        private val slackClient: SlackClient,
+        private val logger: CILogger
+    ) : LintSlackReporter {
+
+        private val tag = "LintSlackAlert"
+
+        override fun report(
+            models: List<LintReportModel>,
+            channel: SlackChannel
+        ) {
+            models.forEach { model ->
+                if (shouldSendAlert(model)) {
```

Comment:
change shouldSend to a `filter { }`

Reply:
i don't want to change it like that; models should gone, as we analyze single report at a time